src/bin/text/dune

```diff
@@ -1,9 +1,18 @@
 (documentation
   (package alt-ergo))
 
+(executable
+  (name gen_link_flags)
+  (libraries unix fmt stdcompat cmdliner)
```

Comment:
It doesn't look like we use `cmdliner`, it should not be a dependency.

Reply:
I wrote a previous version with `cmdliner` but it was a bit over-engineering. I forgot to remove the dependency.